### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by fdallaireCoveo**
I added information in the 3 README file of the project.

### CODE REVIEW

**Comment by fdallaireCoveo** [comment]
@jdevost, when you have a few minutes, please review my PR for the README improvements.

File: misc/Tampermonkey/README.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
-#Tampermonkey Script
-The Tampermonkey script part
+# Tampermonkey Script
 
-## Description
-When you create Web or Sitemap sources in the Coveo platform, you can add Web scraping rules to retrieve or to exclude content from the web pages it will crawl.
-This Tampermonkey script helps with testing your web scraper rules.
-You will be able to test your rules and verify which metadata will be extracted from the page without having to re-index your source.
+The Tampermonkey script is no longer up-to-date. It is recommended to rather use the more convenient and up-to-date Google Chrome extension also included in this repo (see Chrome extension [README](../chrome_extension/README.md)).
```
Comment (jdevost) [inline misc/Tampermonkey/README.md]: > The Tampermonkey script is no longer up-to-date.

Should we use something like "The Tampermonkey script is now deprecated." instead?

I feel like "no longer up-to-date" means we are too lazy update it. And "deprecated" means we won't update it.

File: misc/Tampermonkey/README.md [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
-## How-to Use
+Tampermonkey is a popular userscript manager that you can install in most browsers (see [Tampermonkey](https://tampermonkey.net/)). The web-scraper-helper Tampermonkey script allows you to create and test web scraping configurations directly from the brwoser when visiting web pages for which you want to customize the indexing. 
 
-* Import the `misc/Webscraping.user.js` file in your Tampermonkey extension.
-* From within the Tampermonkey editor:
-  * Edit the `@match` to include the site you are working with
-  * Edit the `jsonRules`
-  * Save the script
-* Reload your site
+The web scraping configuration developed with the extenson can tell the Seb and Sitemap sources to exclude web page sections and extract metadata (see [Web Scraping Configuration](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=277)). The extension does not currently support testing the creation of sub-items.
```
Comment (jdevost) [inline misc/Tampermonkey/README.md]: > tell the Seb and Sitemap sources

Typo: "Seb" --> Web

File: misc/chrome_extension/README.md [hunk 3]
```diff
@@ -1,27 +1,51 @@
 # Chrome Extension
-The chrome extension part
+The web-scraper-helper Google Chrome extension allows you to easily create and test web scraping configurations to use with Coveo Cloud V2 Web and Sitemap source types. 
 
-## Description
-This started out as a tampermonkey script turned into a chrome extension. The goal of this is to make creating and changing the JSON rules for the Coveo webcrawler faster.
+The web scraping configuration developped with the extenson can tell the crawler to exclude web page sections and extract metadata (see [Web Scraping Configuration](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=277)). The extension does not currently support testing the creation of sub-items.
```
Comment (jdevost) [inline misc/chrome_extension/README.md]: 2 typos: developped with the extenson --> developed with the extension

File: misc/chrome_extension/README.md [hunk 4]
```diff
@@ -1,27 +1,51 @@
...
+1. In Google Chrome:
+   1. Go to the **Extensions** page.
+   1. Select the **Developer mode** check box.
+   1. Click **Load unpack extension**. 
+   1. Browse the `chrome_extension` folder of  the repo, and then click **Select**.
+   1. Ensure the **Enabled** check box is selected for the web-scraper-helper extension. 
+
+## Usage
+1. With Google Chrome, go to any web page for which you want to create a web scraping configuration.
+1. Open the Chrome Developer tools (Windows: Control+Shift+I Mac: alt+cmd+I).
```
Comment (jdevost) [inline misc/chrome_extension/README.md]: Maybe have a separator between Windows shortcut and the Mac one?

File: misc/Tampermonkey/README.md (hunk 1 above)
Comment (fdallaireCoveo) [reply misc/Tampermonkey/README.md]: Yes, much better. Fixing it.

**Comment by fdallaireCoveo** [comment]
@jdevost, thanks for the comments. I made the fixes.

File: misc/Tampermonkey/README.md [hunk 5]
```diff
@@ -1,21 +1,29 @@
-#Tampermonkey Script
-The Tampermonkey script part
+# Tampermonkey Script
 
-## Description
-When you create Web or Sitemap sources in the Coveo platform, you can add Web scraping rules to retrieve or to exclude content from the web pages it will crawl.
-This Tampermonkey script helps with testing your web scraper rules.
-You will be able to test your rules and verify which metadata will be extracted from the page without having to re-index your source.
+The Tampermonkey script is now depricated. It is recommended to rather use the more convenient and up-to-date Google Chrome extension also included in this repo (see Chrome extension [README](../chrome_extension/README.md)).
```
Comment (jdevost) [inline misc/Tampermonkey/README.md]: should be `deprecated`.

**Pull request merged**